src/containers/Wallet/index.jsx

```diff
@@ -78,7 +79,7 @@ class Wallet extends React.Component {
 						onKeyPress={() => this.sendRedirect(contractId)}
 					>
 						<div className="balance-info">
-							<span>{FormatHelper.formatAmount(token.get('balance'), token.get('precision'))}</span>
+							<span>{new BN(token.get('balance')).div(10 ** token.get('precision')).toString(10)}</span>
```

Comment:
we need formatAmount for separate every 3 integer symbols like `100 200 000.98989898`

Reply:
need to redo formatAmount